### PR TITLE
Add com.mattjakeman.ExtensionManager

### PR DESCRIPTION
**Pull request by mjakeman**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
    - Uses `--talk-name=org.gnome.Shell.Extensions` to communicate with GNOME Shell to install/update/delete shell extensions.
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/mjakeman/extension-manager/
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)* N/A

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by mjakeman** [comment]
bot, build com.mattjakeman.ExtensionManager

File: com.mattjakeman.ExtensionManager.json [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+                {
+                    "type": "git",
+                    "url": "https://gitlab.gnome.org/GNOME/pango.git",
+                    "tag": "1.50.3",
+                    "commit": "26aadb2508f9022cbfc72e73b558c6791f5d46d9"
+                }
+            ]
+        },
+        {
+            "name": "gtk",
```
Comment (barthalion) [inline com.mattjakeman.ExtensionManager.json:70]: Consider cleaning up pango and GTK binaries/demos from /app/bin/.

**Comment by mjakeman** [comment]
bot, build com.mattjakeman.ExtensionManager

File: com.mattjakeman.ExtensionManager.json (hunk 1 above)
Comment (mjakeman) [reply com.mattjakeman.ExtensionManager.json]: Removed, as well as sassc/blueprint since they aren't needed during runtime.

**Pull request closed without merging**